lib/style.css

```diff
@@ -100,7 +114,13 @@ body {
   padding-top: 0;
 }
 
+/*
+* TODO: Add an option to gmusic-theme.js that handles the difference between
+*       light and dark themes.  The image below should be for "dark" themes only
+*/
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```

Comment:
I was playing around with CSS filters to see if we could get something that works but I couldn't =(

Another option here would be a data URI of our own modified `black_x2.gif`

https://en.wikipedia.org/wiki/Data_URI_scheme